client/src/reducers/formReducer.js

```diff
@@ -93,6 +93,27 @@ const form = formReducer.plugin({
             }
         }
     },
+
+    appChannelFilter: (state, action) => {
+        switch (action.type) {
+            case actions.CHANNELS_LOADED: {
+                const channels = action.payload
+                const filterValues = channels.reduce((acc, c) => {
+                    acc[c.name] = c.name === 'Stable'
+                    return acc
+                }, {})
+                return {
+                    ...state,
+                    values: {
+                        ...filterValues,
```

Comment:
I know I wrote this, but this could just be `values: filterValues`. I was trying to merge the current values in, but that is undefined at this point. And I forgot to change it back.